python/lsst/cp/pipe/ptc/cpExtractPtcTask.py

```diff
@@ -277,24 +277,23 @@ def run(self, inputExp, inputDims):
 
         if self.config.numEdgeSuspect > 0:
             isrTask = IsrTask()
-            self.log.info(f"Masking {self.config.numEdgeSuspect} pixels from the edges "
-                          "of all exposures as SUSPECT.")
+            self.log.info("Masking %d pixels from the edges of all exposures as SUSPECT.",
```

Comment:
I thought it was preferred to use f-strings. Isn't it the case?

Reply:
You aren't allowed to use f-strings in log messages. Log messages are designed to deliberately defer the string formatting until it is known that the message has to be output. If you have warning log messages only then this info message will not appear and the message will never be formatted. With an f-string it always gets formatted.